### PR TITLE
Rewrite and update bundle.properties

### PR DESCRIPTION
**Pull request by SMOLKEYS**
me doing my thing

### CODE REVIEW

File: bundles/bundle.properties [hunk 1]
```diff
@@ -109,12 +109,12 @@ block.heavy-armaments-t1_duo.details = An Upgrade to the popular 'Duo'. Shoots l
...
 block.heavy-armaments-t1_missile.name = Hellseeker
-block.heavy-armaments-t1_missile.description = Simple missile launcher, has a huge range.
+block.heavy-armaments-t1_missile.description = A simple missile launcher which has a lot of range.
 block.heavy-armaments-t1_missile.details = Standard issue surface to air missile launcher. Old and reliable. 
 
 block.heavy-armaments-t2_missile.name = Stiletto
-block.heavy-armaments-t2_missile.description = Fires missiles in short burst
-block.heavy-armaments-t2_missile.details = Many micro missiles were bound together before firing, enabling multiple missiles to be fired at quick succession, although due to this design it loses velocity rather quickly. n/Turret by sh1penfire
+block.heavy-armaments-t2_missile.description = Fires missiles in short bursts.
+block.heavy-armaments-t2_missile.details = Many micro missiles were bound together before firing, allowing it multiple missiles to be fired at quick succession, although due to this design it loses velocity rather quickly. n/Turret by sh1penfire.
```
Comment (Eschatologue) [inline bundles/bundle.properties:117]: This part sounds weird, are you sure?
Thank you for unbutchering my terrible english btw

Comment (SMOLKEYS) [reply bundles/bundle.properties:117]: got confused redoing that part

File: bundles/bundle.properties [hunk 2]
```diff
@@ -1,72 +1,72 @@
 ;/ items
 item.heavy-armaments-cryogel.name = Cryogel
 item.heavy-armaments-cryogel.description = Used as thermal munitions.
-item.heavy-armaments-cryogel.details = An extremely cold compound, synthesized from mixing cyroliquid and sand. Despite its temperature, it is very gelatinous.  Dont eat it
+item.heavy-armaments-cryogel.details = An extremely cold compound, synthesized from mixing cyrofluid and sand. Despite its temperature, it is very gelatinous.  Dont eat it.
```
Comment (Prosta4okua) [inline bundles/bundle.properties:4]: prob cryofluid

**Comment by SMOLKEYS** [comment]
hm.

**Pull request closed without merging**